pkg/acl/acl.go

```diff
@@ -261,7 +261,8 @@ func ApplyAccessControlPolicies(ctx context.Context, operation string, httpVerb
 	}
 
 	action, actionPolicy := IsOperationAllowedByAccessControlPolicy(spiffeID, appID, operation, httpVerb, appProtocol, acl)
-	emitACLMetrics(actionPolicy, appID, trustDomain, namespace, operation, httpVerb.String(), action)
+	emitACLMetrics(ctx,
```

Comment:
I think this should use `context.Background()` so it's not tied to the context of the request. (Like we do in most other places)